photon-colors/photon-colors.swift

```diff
@@ -2,86 +2,91 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/* Photon Colors iOS Variables v3.0.1
+/* Photon Colors iOS Variables v3.1.0
    From https://github.com/FirefoxUX/design-tokens/tree/master/photon-colors#readme */
 
 extension UIColor {
     struct Photon {
-        static let Magenta50 = UIColor(rgb: 0xff1ad9)
-        static let Magenta60 = UIColor(rgb: 0xed00b5)
-        static let Magenta70 = UIColor(rgb: 0xb5007f)
-        static let Magenta80 = UIColor(rgb: 0x7d004f)
-        static let Magenta90 = UIColor(rgb: 0x440027)
+        static let Magenta50 = UIColor(rgb: 0xff1ad9);
+        static let Magenta60 = UIColor(rgb: 0xed00b5);
+        static let Magenta70 = UIColor(rgb: 0xb5007f);
+        static let Magenta80 = UIColor(rgb: 0x7d004f);
+        static let Magenta90 = UIColor(rgb: 0x440027);
```

Comment:
There shouldn't be semicolons here… Did you run an older version of the script?

Reply:
Fix with bccff2e ☺️